scripts/e2e-widget.js

```diff
@@ -48,6 +51,21 @@ async function runNotebook(name) {
   });
 }
 
+function checkIfAllNotebooksHaveTests() {
+  console.log(`Checking if all notebooks under ${baseDir} has tests`);
+  const files = fs
+    .readdirSync(baseDir)
+    .filter((f) => f.startsWith(filePrefix) && f.endsWith(".ipynb"))
+    .map((f) => f.replace(".ipynb", ""));
+  const allNotebooksHaveTests = _.isEqual(_.sortBy(files), _.sortBy(fileNames));
+  if (!allNotebooksHaveTests) {
+    throw new Error(
+      `All notebooks doesn't have tests. If new notebook is added, Please add tests.`

```

Comment:
"All notebooks doesn't have tests" -> "Some of the notebooks don't have tests"


"If new notebook" -> "If a new notebook"

Reply:
Thanks for the comment. Updated the statement.